debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/converters/TinyIntOneToBooleanConverter.java

```diff
@@ -46,7 +50,7 @@ public void configure(Properties props) {
 
     @Override
     public void converterFor(RelationalColumn field, ConverterRegistration<SchemaBuilder> registration) {
-        if (!"TINYINT".equalsIgnoreCase(field.typeName()) || field.length().orElse(-1) != 1 || !selector.test(field)) {
+        if (!TINYINT_FAMILY.contains(field.typeName()) || field.length().orElse(-1) != 1 || !selector.test(field)) {
```

Comment:
@harveyyue Is not here introduced case sensitive comaprison instead of case insesnitive?

Reply:
Actually, we convert the data type name to upper case, however, I fix it in case some code changes.
https://github.com/debezium/debezium/blob/main/debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/antlr/listener/ColumnDefinitionParserListener.java#:~:text=String%20dataTypeName%20%3D%20dataType.name().toUpperCase()%3B